packages/components/src/components/carousel/carousel.ts

```diff
@@ -342,7 +373,11 @@ export default class SdCarousel extends SolidElement {
       canSnap = Math.abs(previousIndex - this.slidesPerMove) % this.slidesPerMove === 0;
     }
 
-    this.goToSlide(previousIndex, behavior);
+    if (this.currentPage - 1 === 0 && this.loop) {
+      this.goToSlide(this.activeSlide - this.slidesPerPage, behavior);
+    } else {
+      this.goToSlide(previousIndex, behavior);
+    }
```

Comment:
<video src="https://github.com/user-attachments/assets/cf0a48bf-02e1-4a04-bf46-1ddea036405e"></video>

This behaviour feels wrong.
- 5 slides, 2 per move, 2 per page
- Current: [4 5] → [3 4] → [1 2]
- Should: [4 5] → [2 3] → [1 2]

In the initial ticket ti says "it should always move the amount of slides per page as described if possible" – which would be currently violated.